### DISCUSSION TITLE
Hover Intellisense in one long line. (no new line is respected)

### DISCUSSION THREAD

**Discussion by Xiaorui-Huang** [microsoft/vscode-cpptools#11663] [Q&A]
Same issue as this closed issue. 
https://github.com/microsoft/vscode-cpptools/issues/2937

although it is marked as fixed, the problem still persists. According to this thread, some are also have the same issue, while others don't 
https://stackoverflow.com/questions/68962181/how-to-show-new-line-in-the-hint-generated-from-comment-in-vs-code

I even tried to use a new **code space** that only have cpp extensions but the problem still persist. 

Can anyone help? Otherwise I may need to reopen the original issue. 
![image](https://github.com/microsoft/vscode-cpptools/assets/42769475/9aedc12a-0a8f-4d8c-aa1e-2aa668e0c75c)
![image](https://github.com/microsoft/vscode-cpptools/assets/42769475/c6313a06-2f44-4d34-ba8e-6dcb3d55ec94)
![image](https://github.com/microsoft/vscode-cpptools/assets/42769475/65ad4fd6-766d-46ff-a1e3-0d9923649b6f)

**Comment by sean-mcmanus** [comment]
This might be bug https://github.com/microsoft/vscode-cpptools/issues/10039 and https://developercommunity.visualstudio.com/t/Avoid-converting-newlines-into-spaces-in/10178887
